frontend/utilities/url/index.ts

```diff
@@ -9,7 +9,6 @@ import {
   HOSTS_QUERY_PARAMS,
   MacSettingsStatusQueryParam,
 } from "services/entities/hosts";
-import vulnerability from "interfaces/vulnerability";
```

Comment:
bug on this branch I committed in a previous PR (should've passed vulnerability through the function but somehow added that variable name as an import >.<